fe/fe-core/src/main/java/com/starrocks/http/rest/TableQueryPlanAction.java

```diff
@@ -189,16 +187,13 @@ private void handleQuery(ConnectContext context, String requestDb, String reques
                              Map<String, Object> result) {
         StatementBase statementBase;
         ExecPlan execPlan;
-        SqlScanner input =
-                new SqlScanner(new StringReader(sql), context.getSessionVariable().getSqlMode());
-        SqlParser parser = new SqlParser(input);
         try {
             /*
              * SingleNodeExecPlan is set in TableQueryPlanAction to generate a single-node Plan,
              * currently only used in Spark/Flink Connector
              */
             context.getSessionVariable().setSingleNodeExecPlan(true);
-            statementBase = SqlParserUtils.getFirstStmt(parser);
+            statementBase = com.starrocks.sql.parser.SqlParser.parse(sql, context.getSessionVariable().getSqlMode()).get(0);
```

Comment:
I think the origin is more readable, why not just change the function SqlParserUtils.getFirstStmt

Reply:
SqlParserUtils.getFirstStmt is the old parser, the modified one is the new parser